### PR TITLE
Add new course type (SEO) and new SEO course

### PR DESCRIPTION
**Pull request by baciucristian**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Added SEO course type and new course (Curso prático de SEO)

### How do we know it's really free?
Because it's on Youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a online course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -356,3 +357,8 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+
+
+### SEO
+
+* [Curso prático de SEO](https://www.youtube.com/playlist?list=PLHz_AreHm4dm4pBTRvBFMpSXvEoymoa90) - Gustavo Guanabara, Curso em Video (YouTube)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:364]:
```suggestion
* [Curso prático de SEO](https://www.youtube.com/playlist?list=PLHz_AreHm4dm4pBTRvBFMpSXvEoymoa90) - Gustavo Guanabara, Curso em Video 
```

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -356,3 +357,8 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+
+
+### SEO
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:362]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Comment (baciucristian) [reply courses/free-courses-pt_BR.md:362]: Added a new course to SEO category.

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -356,3 +357,11 @@
...
+
+* [Curso de SEO - Mão na massa](https://www.youtube.com/playlist?list=PLVPIs-7SxXA_O-fUH5PbKhEHdTPnYKMiE) -
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:364]: single line

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -356,3 +357,11 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+
+
+### SEO
+
+* [Curso de SEO - Mão na massa](https://www.youtube.com/playlist?list=PLVPIs-7SxXA_O-fUH5PbKhEHdTPnYKMiE) - 
+Flavio Klens (Agência Klens)
+
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:366]: no space

**Review by baciucristian — COMMENTED** [review]
Fix spacing

**Pull request merged**